packages/syntax-core/src/Box/Box.tsx

```diff
@@ -418,6 +453,9 @@ const Box = forwardRef<HTMLDivElement, BoxProps>(function Box(
     lgMarginEnd,
     lgMarginStart,
     lgMarginTop,
+    // Overflow
+    overflowX,
+    overflowY,
```

Comment:
@somethiiing are you planning to wire in the `smOverflowX`, `lgOverflowX` ... responsive versions of these props in this PR?

Reply:
i could, but i didnt think `Overflow` was one of those props that needed a sm and lg variation. I could be wrong about this, what do others think?